RelationalAI.Test/DatabaseTest.cs

```diff
@@ -134,29 +135,29 @@ public void DatabaseCloneTest()
             Assert.NotNull(rel);
 
             // make sure the model was cloned
-            var modelNames = client.ListModelNames(databaseCloneName, EngineName);
+            var modelNames = await client.ListModelNamesAsync(databaseCloneName, EngineName);
             var name = modelNames.Find( item => item.Equals("test_model") );
             Assert.NotNull(name);
 
-            var models = client.ListModels(databaseCloneName, EngineName);
+            var models = await client.ListModelsAsync(databaseCloneName, EngineName);
             var model = models.Find( m => m.Name.Equals("test_model") );
             Assert.NotNull(model);
 
-            model = client.GetModel(databaseCloneName, EngineName, "test_model");
+            model = await client.GetModelAsync(databaseCloneName, EngineName, "test_model");
             Assert.NotNull(model);
             Assert.True(model.Value.Length > 0);
 
             // cleanup
-            var deleteRsp = client.DeleteDatabase(databaseCloneName);
+            var deleteRsp = await client.DeleteDatabaseAsync(databaseCloneName);
             Assert.Equal(databaseCloneName, deleteRsp.Name);
         }
 
         public override void Dispose()
         {
             var client = CreateClient();
 
-            try { client.DeleteDatabase(Dbname); } catch {}
-            try { client.DeleteEngineWait(EngineName); } catch {}
+            try { client.DeleteDatabaseAsync(Dbname).Wait(); } catch {}
```

Comment:
we probably should at least do Console.Error here in the catch block so it doesn't get lost?

Reply:
(just noting things, whether we think it's this PR or not)